examples/cli/cli_demo.go

```diff
@@ -58,6 +58,10 @@ func connect() {
 	config.PublishKey = "demo"
 	config.SubscribeKey = "demo"
 	config.SecretKey = "demo"
+	config.PublishKey = "pub-c-7e5c6521-91d0-4e60-9656-4bed419a769b"
```

Comment:
Is there anyway these can be imported from somewhere encrypted?

Reply:
Will think about it. I will remove the keys for now, `demo` key will do the job. And the keys will be deleted.

Reply:
Keys removed.